src/IO/ADIOS/ADIOS2IOHandler.cpp

```diff
@@ -2042,8 +2042,25 @@ ADIOS2IOHandler::ADIOS2IOHandler(
 {
 }
 
-std::future< void >
-ADIOS2IOHandler::flush()
+std::string
+ADIOS2IOHandler::backendProperty( std::string const &property ) const
+{
+    if( property == "sst" )
+    {
+#if defined( __GNUG__ ) || defined( __clang__ )
```

Comment:
This looks a bit empiric - is there an ADIOS2 property/define that we could query for SST capability?

Reply:
I have found no such thing unfortunately. Can ask this tomorrow in the VC though.

Reply:
Sounds good! Yes, one challenge is also that SST support in ADIOS2 depends on the binary variant we build for ADIOS2. So you can get a fully valid ADIOS2 install in some cases that has SST disabled. We might need a runtime API query for that from ADIOS2.